mgr/cephadm: offline host handling for maintenance exit command (#60096)
ceph/ceph: doc/cephadm/host-management.rst

.. prompt:: bash #

   ceph orch host maintenance enter <hostname> [--force] [--yes-i-really-mean-it]
   ceph orch host maintenance exit <hostname> [--force] [--offline]

> The ``--force`` flag on the enter command allows the user to bypass warnings (but not alerts).

Reviewer: ```suggestion
The ``--force`` flag on the ``enter`` command allows the user to bypass warnings (but not alerts).
```

The ``--yes-i-really-mean-it`` flag bypasses all safety checks and will attempt to force the

> host into maintenance mode no matter what. The ``--force`` and ``--offline`` flags on the exit command

Reviewer: ```suggestion
host into maintenance mode. The ``--force`` and ``--offline`` flags to the ``exit`` command
```

can be used to to have cephadm mark a host that is in maintenance mode and offline as no longer

> in maintenance mode. Note in this case if the host was to come back online, the ceph daemons

Reviewer: ```suggestion
in maintenance mode. Note in this case if the host comes online, Ceph daemons
```

> on the host would still be in the stopped state. The ``--force`` and ``--offline`` flags in the exit

Reviewer: ```suggestion
on the host will remain in the stopped state. The ``--force`` and ``--offline`` flags to the ``exit``
```
